src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
ZFS will improve the situation but speaking from experience working here, in the last few years the likelihood of a file in /etc having the wrong contents after a filesystem error is much, much higher than anything else being wrong. In all that time I don't recall ever seen a corrupted config.xml on a system that was still operational. We've seen dozens if not hundreds of systems that needed reinstalled because of filesystem issues that affected files in /etc.